src/main/java/core/basesyntax/Calculate.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Calculate {
```

Comment:
Your name doesn't fully resemble the meaning of the interface, you can use AreaCalculator from check list for example